src/error.rs

```diff
@@ -74,6 +76,7 @@ impl fmt::Display for ExternalError {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> Result<(), fmt::Error> {
         match self {
             ExternalError::NotSupported(e) => e.fmt(f),
+            ExternalError::Ignored => write!(f, "Operation was ignored"),
```

Comment:
Nit:

```suggestion
            ExternalError::Ignored => write!(f, "operation was ignored"),
```

Reply:
We're using upper case letters in this file for display issues, so if we want to change we need to do that all at once.